SPECS/distroless-packages/distroless-packages.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        Metapackage with core sets of packages for distroless containers.
 Name:           distroless-packages
-Version:        0.1
-Release:        3%{?dist}
+Version:        3
```

Comment:
I think we might as well bump it to `3.0` for consistency:
- https://github.com/microsoft/CBL-Mariner/pull/7089